Unittest for graphql have failures.

### Environment
* Python version:  3.8
* Nautobot version:  1.0.1
* nautobot-golden-config version:  0.95

invoke unittest has 2 failures now, looks like when the decorator for graphql was removed from the models to force it to use the custom types that the foreign key references in models is no longer visible form the model.

### Steps to Reproduce
1. git clone
2. invoke unittest


<!-- What did you expect to happen? -->
### Expected Behavior
Successful Unittest

<!-- What happened instead? -->
### Observed Behavior
```
======================================================================
FAIL: test_query_compliance_rule (nautobot_golden_config.tests.test_graphql.TestGraphQLQuery)
Test Configuration Compliance Details Model.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/source/nautobot_golden_config/tests/test_graphql.py", line 287, in test_query_compliance_rule
    self.assertEqual(result.data, response_data)
AssertionError: None != {'compliance_rules': [{'feature': {'name'[117 chars] '}]}

======================================================================
FAIL: test_query_config_compliance (nautobot_golden_config.tests.test_graphql.TestGraphQLQuery)
Test GraphQL Config Compliance Model.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/source/nautobot_golden_config/tests/test_graphql.py", line 226, in test_query_config_compliance
    self.assertEqual(result.data, response_data)
AssertionError: None != {'config_compliances': [{'actual': 'aaa t[162 chars]se}]}

----------------------------------------------------------------------
Ran 28 tests in 2.089s

```
Need to revist if the custom types defined in graphql/types.py are really needed?  Or just need to change the test to remove the `feature` reference in the query as that is not an available parameter to query on. 
```

--- a/nautobot_golden_config/models.py
+++ b/nautobot_golden_config/models.py
@@ -35,6 +35,7 @@ def null_to_empty(val):
     "custom_fields",
     "custom_validators",
     "export_templates",
+    "graphql",
     "relationships",
     "webhooks",
 )

